docs/releases.md

```diff
@@ -0,0 +1,127 @@
+
```

Comment:
nit: Extra whitespace

Reply:
Fixed, thanks.

Reply:
This isn't fixed. I mean there's still an extra line at the top of the file.